### ISSUE TITLE
THIS APP IS DEAD. Use a fork of it

### ISSUE THREAD

**Issue by privacyfreak84** [vfsfitvnm/ViMusic#1492]
This app seems dead. Uou can use forks of it:

RiMusic: https://github.com/fast4x/RiMusic

ViTune: https://github.com/25huizengek1/ViMusic

Music You: https://github.com/DanielSevillano/music-you

Thank you vfsfitvnm for all your work!
I hope that this app continues development

**Comment by asxhelr657** [comment]
Hello, I would like you to tell me more about RiMusic. Does it fix ViMusic bugs? Is it constantly updated? Is it a modification of ViMusic? What are its advantages compared to ViMusic? Is it safe? If I download it, will I lose ViMusic information? Sorry if these are obvious questions, I'm new here.

**Comment by ShareASmile** [comment]
After prolonged use of above RiMusic fork & below fork of ViMusic,

I felt 25huizengek1's fork is better, it is developed consistently, it gives the feel like ViMusic, also it can import ViMusic's export data.

https://github.com/25huizengek1/ViMusic

You can add its custom repo to fdroid app to get easy updates
https://github.com/25huizengek1/ViMusic?tab=readme-ov-file#installation

**Comment by privacyfreak84** [comment]
> Hello, I would like you to tell me more about RiMusic. Does it fix ViMusic bugs? Is it constantly updated? Is it a modification of ViMusic? What are its advantages compared to ViMusic? Is it safe? If I download it, will I lose ViMusic information? Sorry if these are obvious questions, I'm new here.

Yes, RiMusic fixes, in my experience, all of ViMusic's bugs. It is constantly updated (every few days). It is a fork (clone or modification) of ViMusic. It takes ViMusic and builds on top of it to fix all of its bugs and implement new features and quality of life improvements. It is, in my experience, 100% safe. RiMusic is a separate app. When you install it, it will be installed as a separate app from ViMusic. If you want, you can take a backup of all of ViMusic's data (from the settings in ViMusic) and import it all into RiMusic.This will backup all your playlists, artists and stuff. But be aware that this won't import your offline cached songs. With RiMusic, you can also download songs into your device along with having songs cached

**Comment by privacyfreak84** [comment]
> After prolonged use of above RiMusic fork & below fork of ViMusic,
> 
> I felt 25huizengek1's fork is better, it is developed consistently, it gives the feel like ViMusic, also it can import ViMusic's export data.
> 
> https://github.com/25huizengek1/ViMusic
> 
> You can add its custom repo to fdroid app to get easy updates
> https://github.com/25huizengek1/ViMusic?tab=readme-ov-file#installation

The app doesn't even install for me

**Comment by DarkCrypt** [comment]
Both forks (RiMusic & ViMusic) are good. The ViMusic fork needs necessary rebranding which is planned in the near future, it will eventually be called ViTune. The RiMusic fork provides 2 different layouts and is the closest app that works on Android TV but still needs more support.

If you like ViMusic's UI, [Harmony Music](https://github.com/anandnet/Harmony-Music) is basically a Flutter version.

Otherwise, I know of 13 other music streaming apps users can try.

**Comment by bartoostveen** [comment]
> > After prolonged use of above RiMusic fork & below fork of ViMusic,
> > 
> > I felt 25huizengek1's fork is better, it is developed consistently, it gives the feel like ViMusic, also it can import ViMusic's export data.
> > 
> > https://github.com/25huizengek1/ViMusic
> > 
> > You can add its custom repo to fdroid app to get easy updates
> > https://github.com/25huizengek1/ViMusic?tab=readme-ov-file#installation
> 
> The app doesn't even install for me

That is because I still kept ViMusic's application ID, so you can't overwrite ViMusic if you still have vfsfitvnm's build installed. After the rebranding I'll change the application ID.

**Comment by ShareASmile** [comment]
>The app doesn't even install for me

It seems you 
have vfsfitvnm's original ViMusic already installed in your phone & facing [this](https://github.com/25huizengek1/ViMusic/issues/99) issue.

Reason is 25huizengek1's fork till now uses same package ID as vfsfitvnm's ViMusic, however there are plans for Rebranding app from ViMusic to ViTune https://github.com/25huizengek1/ViMusic/issues/135

https://github.com/25huizengek1/ViMusic/discussions/54

Untill then either uninstall original ViMusic, but remember exporting your data first
OR
if you don't want to uninstall it, then better use nightly by adding repo to fdroid app from [this link](https://25huizengek1.github.io/fdroid-repo/fdroid/repo/) or download directly from [github nightly repo](https://github.com/25huizengek1/fdroid-repo/blob/master/fdroid/repo/app.vitune.android.nightly_20240902.apk)

nightly version can be installed as a separate app & can receive regular early updates through fdroid repo

**Comment by bartoostveen** [comment]
While I politely invite people who are facing further issues with my fork, I do want to mention:

I think it's ridiculous that there is a separate issue straight-up advertising RiMusic. Sometimes some of my users or myself mention that some bug is fixed in my fork, but that is just a suggestion in order to resolve an upstream issue ticket. Advertising repositories in this way, using a separate ticket, is just unnecessary.

**Comment by siggi1984** [comment]
Check out this ViMusic fork. 
I think it works quite well. Simple and functional.

https://github.com/DanielSevillano/music-you

**Comment by privacyfreak84** [comment]
> Check out this ViMusic fork.
> I think it works quite well. Simple and functional.
> 
> https://github.com/DanielSevillano/music-you

Very slow and laggy for me

**Comment by devcomputer** [comment]
i can not export liked artists, songs and albums.

**Comment by privacyfreak84** [comment]
> i can not export liked artists, songs and albums.

In which app?

**Comment by devcomputer** [comment]
> > i can not export liked artists, songs and albums.
> 
> In which app?

ViMusic.apk

**Comment by siggi1984** [comment]
Check out this fork of ViMusic. It's very well done. Not too overloaded and keeps the simplicity of ViMusic, unlike some other forks.

https://github.com/25huizengek1/ViTune
